apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -502,23 +488,7 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
       // End modal.
       onClose();
     }
-  };
-
-  const getPreviewCode = (currentGeneratedEffect?: GeneratedEffect): string => {
-    if (!currentGeneratedEffect) {
-      return '';
-    }
-
-    const tempWorkspace = new Workspace();
-    const previewCode = generatePreviewCode(
-      tempWorkspace,
-      currentGeneratedEffect
-    );
-    tempWorkspace.dispose();
-    return previewCode;
-  };
-
```

Comment:
This was moved to `utils/getPreviewCode` and is a candidate for further refactoring since it's always called with the result of `getGeneratedEffect` and maybe those should be combined.